controller/setting_controller.go

```diff
@@ -914,6 +922,38 @@ func (sc *SettingController) updateKubernetesClusterAutoscalerEnabled() error {
 	return nil
 }
 
+func (sc *SettingController) cleanupShareManagerServiceAndEndpoints() error {
+	var err error
+	defer func() {
+		if err != nil {
+			err = errors.Wrapf(err, "failed to cleanup share manager service and endpoints for %s setting update", types.SettingNameStorageNetworkForRWXVolumeEnabled)
+		}
+	}()
+
+	shareManagers, err := sc.ds.ListShareManagers()
+	if err != nil {
+		return err
+	}
+
+	for _, shareManager := range shareManagers {
+		_log := sc.logger.WithField("shareManager", shareManager.Name)
```

Comment:
Why not just use `log`?

Reply:
It is to improve readability indicating that this is used within the loop. However, I am also fine using `log`.

Updated.